src/settings.ts

```diff
@@ -13,12 +13,13 @@ export interface Settings {
 export interface SentryProject {
     name: string
     projectId: string
-    patternProperties: {
-        repoMatches?: RegExp[]
-        fileMatches: RegExp[]
-        // RexExp patterns to match log handeling code, e.g. /log\.(Printf|Print)\(['"]([^'"]+)['"]\)/
-        lineMatches: RegExp[]
-    }
+    // RexExp patterns to match log handeling code, e.g. /log\.(Printf|Print)\(['"]([^'"]+)['"]\)/
+    linePatterns: RegExp[]
+    filters: {
+        repository?: RegExp[]
+        file?: RegExp[]
```

Comment:
What's the benefit of these being an array?

Reply:
The user doesn't have to create a filter for each repo/file specification, create multiple repo/filters that are valid together.